### PR TITLE
[CB-14140] Remove shelljs

### PR DESCRIPTION
**Pull request by raphinesse**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,8 +22,10 @@
   },
   "dependencies": {
     "cordova-common": "^2.2.0",
+    "fs-extra": "^6.0.1",
     "q": "^1.4.1",
-    "shelljs": "^0.7.0"
+    "shelljs": "^0.7.0",
```
Comment (dpogue) [inline package.json]: Are we able to remove shelljs entirely from the package.json?

Comment (raphinesse) [reply package.json]: Of course. Good catch. Will update this to remove it.

Embarrassing oversight given the title of the PR :sweat_smile:

**Comment by brodybits** [comment]
Linking this wonderful change in progress to Apache CB-14140 (<https://issues.apache.org/jira/browse/CB-14140>).

**Comment by raphinesse** [comment]
@dpogue Are you fine with me merging this?

**Pull request merged**